usaspending_api/accounts/urls_federal_accounts_v2.py

```diff
@@ -13,7 +13,7 @@
 urlpatterns = [
     url(r"(?P<pk>[0-9]+)/available_object_classes$", object_class_federal_accounts),
     url(r"(?P<pk>[0-9]+)/fiscal_year_snapshot/(?P<fy>[0-9]+)/?$", fiscal_year_snapshot_federal_accounts),
-    url(r"(?P<pk>[0-9]+)/fiscal_year_snapshot$", fiscal_year_snapshot_federal_accounts),
+    url(r"(?P<pk>[0-9]+)/fiscal_year_snapshot/?$", fiscal_year_snapshot_federal_accounts),
```

Comment:
Why was this added? Should we be adding this for everything?

Reply:
It was the only endpoint that didn't work with a slash on the end, so I added support.